app/helpers/api/integrations_helper.rb

```diff
@@ -69,11 +65,6 @@ def deployment_option_is_service_mesh?(service)
     service.deployment_option =~ /^service_mesh/
   end
 
-  def edit_deployment_option_title(service)
```

Comment:
Can this like finally be removed as well https://github.com/3scale/porta/blob/80daaa207a1974888083a2bab96a2ed6168290b0/app/views/api/integrations/apicast/configuration_driven/_apicast.html.slim#L3?

Reply:
I missed that one, thanks!